workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -52,7 +70,7 @@ def __repr__(self):
 
 
 def plot_2d_matplotlib(
-    run_diags: RunDiagnostics, varfilter: str, dims: Sequence = None, **opts
+    run_diags: RunDiagnostics, varfilter: str, dims: Sequence, **opts
```

Comment:
cleaned up this function a bit, it had some vestiges of the original `plot_2d` Holoviews function.